README.md

```diff
@@ -7,9 +7,33 @@ This project is a basic local server for the [Riff Racer](https://store.steampow
 - Python 3.6+
 - Flask
 
+(Or Docker)
+
 ## Usage
 
-Run `sed -i "s/https:\/\/parseapi.back4app.com/http:\/\/127.000000000.0.1:5144/g" ParseOctane.dll` in your game directory to patch the Parse API DLL to use RiffApi Local instead. Then, run `riffapi-local.py` and launch the game.
+### Patching ParseOctane.dll
+
+Patch `ParseOctane.dll` to use RiffApi Local changing "https://parseapi.back4app.com" by "http://127.000000000.0.1:5144".
+
+1. Open Steam and go to your game library
+2. Right click on "Riff Racer" on the game list
+3. Press on "manage" > "browse local files", it will open the folder where the game files are located.
+4. Open a command prompt, `shift` + `right-click` and "Open PowerShell window here
```

Comment:
Have you tried whether this works on vanilla Windows? Iirc sed didn't work natively out of box

Reply:
It worked on my pc, checking in more detail seems its due to WSL been installed. I'll change it to Windows native commands

Reply:
I have not been able to find a native Windows command to patch the dll. Windows is changing the end of lines with `CRLF` and the dll uses `CR` (Macintosh) line endings corrupting the file and not launching the game.

What would you think about adding an html file and patching it with js? The same as on your website.

PS, this is the windows command `(get-content ParseOctane.dll) -replace "https://parseapi.back4app.com","http://127.000000000.0.1:5144" | set-content ParseOctane.dll`